tsparser/src/app/mod.rs

```diff
@@ -92,6 +93,29 @@ impl AppValidator<'_> {
     }
 
     fn validate_apis(&self) {
+        let mut seen = std::collections::HashMap::new();
+        for bind in &self.parse.binds {
```

Comment:
I think we should do this based on resources, not binds

Reply:
we can, but then we dont have a range that point to the identifier. We could add that ofc